source/manual/send-test-email.html.md

```diff
@@ -0,0 +1,25 @@
+---
+owner_slack: "#email"
+title: Send a test email via Notify
+section: Development VM
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2017-11-1
```

Comment:
Should this be `-01` instead of `-1` at the end? It might complain if it's not two-digits.